tests/cggi/boolean_line_vectorizer.mlir

```diff
@@ -0,0 +1,75 @@
+// RUN: heir-opt --cggi-boolean-line-vectorize %s
+
+#encoding = #lwe.unspecified_bit_field_encoding<cleartext_bitwidth = 1>
+!ct_ty = !lwe.lwe_ciphertext<encoding = #encoding>
+!pt_ty = !lwe.lwe_plaintext<encoding = #encoding>
+
+// CHECK-LABEL: add_one
+// CHECK-COUNT-1: cggi.packed %[[*]], %[[*]] {gates = #cggi.cggi_gate<"and", "xor">} : (tensor<2x!lwe.lwe_ciphertext
```

Comment:
I feel it would be worth adding a few more assertions for the rest of the program.